src/deltacode/cli.py

```diff
@@ -50,11 +50,36 @@ def write_json(deltacode, outfile, all_delta_types=False):
         ('deltas_count', len([d for d in deltas(deltacode, all_delta_types)])),
         ('delta_stats', deltacode.stats.to_dict()),
         ('deltas', deltas(deltacode, all_delta_types))
-    ])
+    ])  
+
+    try:
+
+        simplejson.dump(results, outfile, iterable_as_array=True, indent=2)
+        outfile.write('\n')    
+        if outfile.name != '<stdout>' : 
+            "This condition arises when we are required to print in the json file and the value of -j in not the console terminal"
+            click.secho(get_notice_for_cli(),fg="green")  
+            click.echo('deltas_count: {}'.format(len([d for d in deltas(deltacode, all_delta_types)])))
+            click.echo('delta_stats: ')
+            for stat in deltacode.stats.to_dict():
+                click.echo("""   "{}": {}""".format(str(stat),deltacode.stats.to_dict()[stat]))  
+            # TODO: add toggle for pretty printing
+
+            delta_count = 1
+            for delta in deltacode.deltas :
+                if delta.status != "unmodified":
+                        click.echo("\n")
+                        click.secho("delta {}".format(delta_count))
+                        click.echo(delta)
+                        delta_count += 1
+            click.secho("#" * 211 + "\n",fg="yellow")
+            click.secho(" "*70 + "All deltas are generated, view the json file for details" + "\n" , fg="green")
+            click.secho("#" * 211,fg="yellow")
```

Comment:
Please remove this. I dont really care for this output.

Reply:
@majurg , I will be removing the `notice` and this additional output sections